src/common/Coverage.ts

```diff
@@ -21,12 +21,67 @@ import { CDPSession } from './Connection';
 
 import { EVALUATION_SCRIPT_URL } from './ExecutionContext';
 
+/**
+ * The CoverageEntry class represents one entry of the coverage report.
+- `url` <[string]> StyleSheet URL
+- `text` <[string]> StyleSheet content
+- `ranges` <[Array]<[Object]>> StyleSheet ranges that were used. Ranges are sorted and non-overlapping.
+  - `start` <[number]> A start offset in text, inclusive
+  - `end` <[number]> An end offset in text, exclusive
```

Comment:
there's no need to document these here - the comments you've added below will do the trick.

What you do need is to mark this as `@public` and also `export` the interface too.

Reply:
Done.